packages/server/events-processing-platform/domain/organization/aggregate/commands.go

```diff
@@ -2,6 +2,8 @@ package aggregate
 
 import (
```

Comment:
missing changes for in aggregate.go file

Reply:
what do you mean?